app-frontend/src/app/pages/market/market.html

```diff
@@ -0,0 +1,2 @@
+<rf-model-search on-search="$ctrl.search(text)"></rf-model-search>
```

Comment:
I'm not sure this is hooked up quite right; it looks like it should be `$ctrl.onSearch(text)` based on what's in `market.controller.js`.